packages/ui/src/components/va-data-table/hooks/useSortable.ts

```diff
@@ -1,6 +1,12 @@
 import { computed, ref, Ref, watch } from 'vue'
 
-import { DataTableColumnInternal, DataTableRow, DataTableItem, DataTableSortingOrder } from '../types'
+import {
```

Comment:
you can try to take adventage of [that](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html)